kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -173,6 +173,8 @@
   import MeteredConnectionNotificationModal from 'kolibri-common/components/MeteredConnectionNotificationModal.vue';
   import appCapabilities, { checkCapability } from 'kolibri.utils.appCapabilities';
   import LearningActivityChip from 'kolibri-common/components/ResourceDisplayAndSearch/LearningActivityChip.vue';
+  import { searchKeys } from 'kolibri-common/composables/useBaseSearch';
```

Comment:
Seems like this addition and removal from the useSearch composable is the only change we need in this file.